packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -162,20 +141,19 @@ export default function MongoNav({
   mode = Mode.Production,
   showProjNav = true,
   admin = false,
-  isOnPrem = false,
   hosts: hostsProp,
   urls: urlsProp,
   constructOrganizationURL: constructOrganizationURLProp,
   constructProjectURL: constructProjectURLProp,
   onError = () => {},
   onSuccess = () => {},
   onLogout = () => {},
-  onPremVersion,
-  onPremMFA = false,
+  onPrem = { mfa: false, enabled: false, version: '' },
 }: MongoNavInterface) {
+  const { mfa, enabled, version } = onPrem;
```

Comment:
I may argue to keep these props on the `onPrem` object to keep the context of their values nearby. I.e. on line 270 it's a little confusing what the enabled variable relates to.

Also should probably change the child props as well to keep them consistent.

Reply:
👍